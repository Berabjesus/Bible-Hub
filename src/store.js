import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from './reducers';

const middlewares = [thunk];
const store = createStore(rootReducer, applyMiddleware(...middlewares));

export default store;
