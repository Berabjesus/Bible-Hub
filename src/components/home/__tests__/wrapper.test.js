import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../store';
import Wrapper from '../wrapper';

describe('Bible page', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <Wrapper />
        </Provider>
      </BrowserRouter>,
    );
    expect(component).toMatchSnapshot();
  });
});
