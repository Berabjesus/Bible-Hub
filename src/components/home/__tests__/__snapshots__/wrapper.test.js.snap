// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bible page matches the snapshot 1`] = `
<div>
  <div
    className="row align-items-start"
  />
</div>
`;

exports[`Wrapper page matches the snapshot 1`] = `
<div>
  <div
    className="row align-items-start"
  />
</div>
`;
